pkgs/development/python-modules/platformio/default.nix

```diff
@@ -23,6 +23,13 @@ buildPythonPackage rec {
 
   patches = [ ./fix-searchpath.patch ];
 
+  postInstall = ''
```

Comment:
Why is this needed? `wrapPythonPrograms` already sets `PATH` and injects `PYTHONPATH`.
Note also that the `postFixup` (or `fixupPhase` in general) is preferred for wrappers.

Reply:
you are right its no longer needed with the other fixes in place. earlier i was not seeing the lru getting linked in properly.